default/specials.txt

```diff
@@ -339,7 +339,7 @@ Special
             scope = Source
             activation = OwnedBy affiliation = AnyEmpire
             effects = [
-                SetDetection value = Value + 75
+                SetDetection value = Value + Min(Target.Construction / 30, 1) * 75
```

Comment:
Ooh, I like this idea, you always get the 75 but might get more if well developed.